### DISCUSSION TITLE
Pause

### DISCUSSION THREAD

**Discussion by alabotski** [iSoron/uhabits#1119] [Feature Requests] [answered]
Can you add something like a pause?

For example, I go to the gym 3 times a week.
But now I am sick and have to miss a couple of classes, but these absences are not my fault and I do not want to lose progress

**Comment by hiqua** [comment, marked as answer]
You can skip days in v2.0, that should fit your need, right?

**Reply by alabotski** [reply]
My version 2.0.3, can you explain me, how use this functional?

**Reply by hiqua** [reply]
Go to settings, enable skip days.

**Comment by alabotski** [comment]
Ok thanks ))
